presto-spi/src/main/java/com/facebook/presto/spi/ConnectorHandleSerde.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.spi;
+
+public interface ConnectorHandleSerde<T>
+{
+    byte[] serialize(T val);
+
+    T deSerialize(Class<? extends T> connectorHandleClass, byte[] byteArr);
```

Comment:
- we don't use abbreviations in the codebase: `value` and `byteArray` (or simply `bytes`)
- deSerialize -> deserialize

Same for all other places in the PR

Reply:
done